code/modules/mob/living/carbon/human/life.dm

```diff
@@ -33,8 +33,11 @@
 
 		else //Dead
 			dead_ticks ++
-			if(dead_ticks > TIME_BEFORE_DNR)
-				set_undefibbable()
+			var/mob/dead/observer/related_ghost = get_ghost()
```

Comment:
Not sure I understand why you declare a var

Reply:
How is it not needed? Did you read every line I changed? I can't access the var without typecasting it and the var just below is used as an arg in set_undefibbable